addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java

```diff
@@ -768,6 +774,43 @@ else if (!contentType.startsWith("image")) {
     }
 
     /**
+     * Save config file send by editor
+     *
+     * @param request
+     * @param response
+     * @throws ServletException
+     * @throws IOException
+     */
+    private final void saveConfig(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String fileName = request.getParameter("config");
+        File file = new File(userFileFolder, URLDecoder.decode(fileName, "UTF-8"));
```

Comment:
Should we check if the absolute path of file starts with the abosulte path of userFileFolder?